src/main/java/net/sf/jsqlparser/util/TablesNamesFinder.java

```diff
@@ -244,6 +244,7 @@ public void visit(IntegerDivision division) {
 
     @Override
     public void visit(DoubleValue doubleValue) {
+    // intentional (commented) empty methods.
```

Comment:
Which PMD rule forces this?

Reply:
I would prefer it without any comment. The name itself is documentation enough.

Reply:
The rule `<rule ref="category/java/documentation.xml/UncommentedEmptyMethodBody" />` is applied by Codacy and will reject any commit affecting this class.
Although I can replace it by a Class wide Annotation.

Reply:
Damned. You are right. However, I will remove it from Codacy.

Reply:
Removed.

Reply:
Yes, but I have solved it before already.
I have removed these comments already and you just need to accept the PR pls.